packages/manager/apps/web/client/app/components/hosting-domain-offers/template.html

```diff
@@ -0,0 +1,64 @@
+<!--Hosting offers-->
+<div class="row">
+    <!--Hosting available offers-->
+    <oui-select-picker
+        class="d-inline-block col-sm-12 col-md-4 my-3"
+        data-ng-repeat="offer in $ctrl.groupedOffers track by $index"
+        data-id="hosting-offer-model-{{offer.category}}"
+        data-name="hosting-offer-model-{{offer.category}}"
+        data-model="$ctrl.model.offer"
+        data-values="[offer]"
+        data-variant="light"
+        data-on-change="$ctrl.onOfferClick(offer)"
+    >
+        <!--Offer picker header-->
+        <oui-select-picker-label class="w-100 d-flex justify-content-between">
+            <!--Offer name-->
+            <div class="m-0">
+                <span
+                    class="font-weight-bold"
+                    data-ng-bind="('web_components_hosting_domain_offers_offer_' + offer.category | translate) | uppercase"
+                ></span>
+            </div>
```

Comment:
**nitpick:** `div` + `span` can be combined into `p`.
```suggestion
            <p class="m-0"
                    class="font-weight-bold"
                    data-ng-bind="('web_components_hosting_domain_offers_offer_' + offer.category | translate) | uppercase"
                >
            </p>
```

Reply:
changed